microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -21,6 +22,14 @@ class CloudHypervisorTestResult:
     status: TestStatus = TestStatus.QUEUED
 
 
+@dataclass
+class CHPerfMetricTestResult:
```

Comment:
The result class doesn't help now. Its values are used in the same method only.

Reply:
Anirudh suggested to implement class for the result.

Reply:
In previous code, there is a separated loop. So, it needs an object to save multiple values together. But for now, there is no separated loop. From the code, values of an object are declared and used in the same loop. That can be done by variables directly.

Reply:
Sure Chi. Will make this change

Reply:
The updated code is the same as before. It doesn't need a dict. If you search the code, you will find the life cycle of the dict is in the for loop only. So, use those variables directly, instead of a dict.

Reply:
Sure Chi